src/ext_manifest.c

```diff
@@ -97,7 +97,7 @@ static int ext_man_build(const struct module *module,
 		fprintf(stderr,
 			"error: failed to read %s section content, code %d\n",
 			EXT_MAN_DATA_SECTION, ret);
-		goto out;
+		return ret;
```

Comment:
This change is extremely weird given the commit message.

Reply:
Also unneeded, since you'd call free(NULL) which is completely safe to do. So unneeded even if correct actually.

Reply:
The code looks correct (functionally the same if we have the ```goto``` or the ```return``` here), I agree I'm a little confused by the commit message.